pom.xml

```diff
@@ -377,9 +377,17 @@
                     <exclude>org.sl4fj:slf4j-jcl</exclude>
                     <exclude>log4j:log4j</exclude>
                     <exclude>ch.qos.logback:logback-classic</exclude>
+                    <!-- NOTE: We're banning any HBase deps versions other than the approved ${hbase.version},
+                               which is aimed at preventing the classpath collisions w/ transitive deps usually) -->
+                    <exclude>org.apache.hbase:hbase-common:*</exclude>
```

Comment:
@yihua : can you review hbase related changes. mostly should be good, just wanted to double confirm.

Reply:
LGTM.  In terms of the actual impact, does this only resolve issues for Spark 3 due to transitive dependencies of Hive?  I assume without this change, Spark 2 should still work.

Reply:
These should resolve issues for all Spark versions